.env.example

```diff
@@ -0,0 +1,7 @@
+DB_HOST=<Database Host>
+DB_PORT=<Database Port>
+DB_USER=<Database Username>
+DB_PWD=<Database Password>
+PORT=<NodeJs app. PORT>
+APP_ENV=<Application environtment: local, development, production>
+SECRET_KEY=<secretKey>
```

Comment:
Change to JWT_SECRET_KEY

Reply:
Fixed

Reply:
https://github.com/EasyEventV2/ee2_api/commit/73716262368593e9f1d6f9ac94ab472189fd237b